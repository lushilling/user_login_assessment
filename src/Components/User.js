import React from 'react';
import '../App.css';

export default class User extends React.Component {
    render(props) {
        return (
            <tr>
                <td>
                    <h4>{props.username}</h4>
                    <p>{props.email}</p>
                </td>
            </tr>
        );
    }
}