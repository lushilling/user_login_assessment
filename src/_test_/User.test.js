import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import User from '../Components/User';


it('renders user', () => {
    const tree = renderer.create(<User />).toJSON()
    expect(tree).toMatchSnapshot();
})

it('renders user without crashing', () => {
    shallow(<User />);
});