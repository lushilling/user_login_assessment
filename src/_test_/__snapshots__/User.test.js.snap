// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders user 1`] = `
<tr>
  <td>
    <h4 />
    <p />
  </td>
</tr>
`;
